tests/conftest.py

```diff
@@ -0,0 +1,76 @@
+import json
+import pytest
+import aioredis
+import asynctest
+
+import pyslackersweb
+import pyslackersweb.website.tasks
+
+pytest_plugins = ("slack.tests.plugin",)
+
+
+def pytest_addoption(parser):
```

Comment:
I don't think it's a good idea to _optionally_ run tests with different configurations, I'd remove this + the signal muddling + "FakeRedis".

Reply:
I'm happy about removing all that. It's true that the `FakeRedis` stuff isn't super nice. It was the only way to bring some tests without a big refactoring.

I would still keep the command line argument to skip those tests if redis and postgresql are not available. Or maybe run the tests with docker-compose each time :thinking: